go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/cch123/elasticsql v1.0.1
 	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
 	github.com/davecgh/go-spew v1.1.1
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
This +incompatible hasn't caused any potential issues, right?

Reply:
I haven't encountered any issues. And `dgrijalva/jwt-go` is a very popular library for handling JWTs.